io/disk/tiobench.py.data/tiobench.yaml

```diff
@@ -0,0 +1,33 @@
+setup: 
+    location: !mux
+        default:
+            target: '/mnt'
+
+    block: !mux
+        first:
+            blocks: 4096
+        second:
+            blocks: 8192
+
+    thread: !mux
+        default:
+            threads: 10
+        minimal:
+            threads: 5
+
+    blocksize: !mux
+        default: 
+            size: 1024
+     
+    runs: !mux
+        default: 
+            numruns: 2
+        minimal:
+            numruns: 1
+  
+    iteration: !mux
```

Comment:
Would this work for you? I mean if you want to evaluate the differences inside the test, then you have to create iterations. In case you just want to store the results and eventually compare them later, the multiplexer is the preferred solution.